src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -120,24 +120,24 @@ public String getApiUri() {
     @Override public void visitSources(SCMSourceObserver observer) throws IOException, InterruptedException {
         TaskListener listener = observer.getListener();
         if (repoOwner.isEmpty()) {
-            listener.getLogger().format("Must specify user or organization%n");
-            return;
+            throw new AbortException("Must specify user or organization%n");
```

Comment:
> Which `FormValidation` should already be catching.

As you well know, this input data validation is a bit poor because allows to the users saving the information even if there are errors.

> The problem here is that `repoOwner` will be blank for a newly created folder—a normal condition.

In this case, the user will see the same comment (because I'm using the same comment in the `AbortException` but the indexing process will be marked as failure. IMO, the expected behavior.